app/code/Magento/Multishipping/Model/Cart/CartTotalRepositoryPlugin.php

```diff
@@ -33,21 +33,21 @@ public function __construct(
     }
 
     /**
-     * Overwrite the CartTotalRepository quoteTotal and update the shipping price
+     * Check multishipping after get cart total
      *
-     * @param                                         CartTotalRepository $subject
-     * @param                                         Totals              $quoteTotals
-     * @param                                         String              $cartId
-     * @return                                        Totals
-     * @throws                                        \Magento\Framework\Exception\NoSuchEntityException
+     * @param CartTotalRepository $subject
+     * @param Totals $result
+     * @param int $cartId
+     * @return Totals
+     * @throws \Magento\Framework\Exception\NoSuchEntityException
      * @SuppressWarnings(PHPMD.UnusedLocalVariable)
      * @SuppressWarnings(PHPMD.UnusedFormalParameter)
      */
     public function afterGet(
         CartTotalRepository $subject,
-        Totals $quoteTotals,
-        String $cartId
-    ) {
+        Totals $result,
```

Comment:
Is there any reason for replacing `quoteTotals` with `result`?

Reply:
i just follow this guide https://devdocs.magento.com/guides/v2.3/extension-dev-guide/plugins.html with after plugin need result parameter. Beside result variable will be more clearly i think

Reply:
yeah i reverted variable back as your catch